src/js/cc_script.js

```diff
@@ -5,139 +5,139 @@
  * Date 09.12.11
  */
 
-"use strict";
+'use strict';
 
 (function (window, $) {
 
-  $.fn.ccCarousel = function (options) {
-
-    options = $.isPlainObject(options) ? options : {};
-
-    /*================================Configurations==============================*/
-
-    var defaults = {
-      slider: $('.slider'), //dynamic div
-      sliderArray: $('.slider div'), //array of images in the slider
-      wrapperWidth: $('.galleryWrapper').width(), //initializing the wrapper gallery width
-      dynamicWidth: $('.dynamic').width(), //initializing the dynamic wrapper width
-      sliderLength: $('.slider div').length // count of images in the slider
+    $.fn.ccCarousel = function (options) {
+
+        options = $.isPlainObject(options) ? options : {};
+
+        /*================================Configurations==============================*/
+
+        var defaults = {
+            slider: $('.slider'), //dynamic div
+            sliderArray: $('.slider div'), //array of images in the slider
```

Comment:
@rhanesoghlyan `$('.slider div')`  the same selector used more than once, please replace with reference from some variable.